include/lsst/jointcal/ConstrainedPhotometryModel.h

```diff
@@ -77,11 +77,18 @@ class ConstrainedPhotometryModel : public PhotometryModel {
 private:
     PhotometryMappingBase *findMapping(CcdImage const &ccdImage) const override;
 
+    // Which components of the model are we fitting?
+    bool _fittingChips;
+    bool _fittingVisits;
+
+    // The per-ccdImage transforms, each of which is a composition of a chip and visit transform.
     typedef std::unordered_map<CcdImageKey, std::unique_ptr<ChipVisitPhotometryMapping>> MapType;
-    MapType _myMap;
+    MapType _chipVisitMap;
 
+    // The per-visit transforms that go into _chipVisitMap.
     typedef std::map<VisitIdType, std::shared_ptr<PhotometryMapping>> VisitMapType;
     VisitMapType _visitMap;
+    // The per-chip transforms that go into _chipVisitMap.
```

Comment:
So is `_chipVisitMap` an optimization to avoid recomputing the composition? It might be good to explicitly spell out the self-consistency requirements (which I assume are enforced by `ConstrainedPhotometryModel`).